src/platform/packages/shared/kbn-discover-utils/src/field_constants.ts

```diff
@@ -14,7 +14,15 @@ export const LOG_LEVEL_FIELD = 'log.level';
 export const MESSAGE_FIELD = 'message';
 export const ERROR_MESSAGE_FIELD = 'error.message';
 export const EVENT_ORIGINAL_FIELD = 'event.original';
+export const EVENT_OUTCOME_FIELD = 'event.outcome';
+
+// Trace fields
 export const TRACE_ID_FIELD = 'trace.id';
+export const TRANSACTION_NAME_FIELD = 'transaction.name';
+export const TRANSACTION_DURATION_FIELD = 'transaction.duration.us';
+export const SPAN_NAME_FIELD = 'span.name';
+export const SPAN_DURATION_FIELD = 'span.duration.us';
+export const DURATION_FIELDS: readonly string[] = [SPAN_DURATION_FIELD, TRANSACTION_DURATION_FIELD];
```

Comment:
There is already a package containing APM fields (https://github.com/elastic/kibana/tree/main/x-pack/platform/packages/shared/kbn-apm-types). Do we need to duplicate them here?

Reply:
I think because it's in `x-pack` but Discover is in `packages`, it can't reference private/premium packages? Unless that is moved out of `x-pack`...